base_url/models/model.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+from openerp import models, api, fields, _
+import logging
+from openerp.exceptions import Warning as UserError
+_logger = logging.getLogger(__name__)
+try:
+    from slugify import slugify
+except ImportError:
+    _logger.debug('Cannot `import slugify`.')
+
+
+class UrlUrl(models.Model):
+
+    _name = "url.url"
+
+    url_key = fields.Char(string="Url Id", store=True)
+    model_id = fields.Reference(selection='_reference_models',
+                                help="The id of product or category.",
+                                readonly=True,
+                                )
+    redirect = fields.Boolean('Redirect')
+
+    _sql_constraints = [("urlurl unique key",
+                         'unique(url_key)',
+                         'Already exists in database')]
+
+    @api.model
+    def _reference_models(self):
+        return []
+
+    @api.multi
+    def _get_object(self, urls):
```

Comment:
urls est un itérable (list, etc) ?
- si oui ("url_key", "in", urls)
- si non s / urls / url